src/Documentation/UserManual/DeviceGenericSerial.dox

```diff
@@ -6,7 +6,8 @@ It does not collect tracking or image data, only allows sending and receiving da
 using OpenIGTLink commands (see \ref PlusServerCommands).
 
 A (customizable) newline string is appended after each sent command and the device listens
-for reponse that is terminated by the same newline string.
+for reponse that is terminated by the same newline string by default.
+The default can be modified by specifying "acceptReply" parameter of ReplyTermination type.

```

Comment:
Maybe I misunderstood Andras' sentence "Please make the AcceptReply criteria configurable and update DeviceGenericSerial.dox"? Or maybe he misunderstood that for a [device](http://www.velmex.com/Downloads/Spec_Sheets/VXM%20-%20%20Command%20Summary%20Rev%20B%20814.pdf) we are dealing with, both settings must be supported within a same session - replies to some commands are CR terminated while a replies to a few commands are not terminated.